CHANGELOG/CHANGELOG-2.4.md

```diff
@@ -0,0 +1,80 @@
+# Changelog 2.4
+
+## 2.4.0 - 2025-07-04
+
+### Added
+
+- Add BGP confederation support in BGPPolicy. ([#6927](https://github.com/antrea-io/antrea/pull/6927) [#6905](https://github.com/antrea-io/antrea/pull/6905), [@hongliangl])
+- Support mTLS when exporting flows to an external flow collector for FlowAggregator. ([#7212](https://github.com/antrea-io/antrea/pull/7212), [@antoninbas])
+- Add `k8s.v1.cni.cncf.io/network-status` annotation to make SecondaryNetwork Pod IP visible. ([#7069](https://github.com/antrea-io/antrea/pull/7069), [@wenqiq])
+- Add `protocolFilter` config to FlowExporter to filter and export flows only with the specified protocols. ([#7145](https://github.com/antrea-io/antrea/pull/7145), [@petertran-avgo])
+- Add `antctl get fqdncache` sub-command to fetch the DNS mapping entries for FQDN policies. ([#6868](https://github.com/antrea-io/antrea/pull/6868), [@Dhruv-J])
+- add TCP flags filter support for PacketCapture. ([#7070](https://github.com/antrea-io/antrea/pull/7070), [@AryanBakliwal])
+- Add bidirectional packet capture support for PacketCapture. ([#6882](https://github.com/antrea-io/antrea/pull/6882), [@AryanBakliwal])
+- Add ICMP messages filter support for PacketCapture. ([#7164](https://github.com/antrea-io/antrea/pull/7164), [@AryanBakliwal])
+- Support `antctl packetcapture` sub-commands for PacketCapture. ([#6884](https://github.com/antrea-io/antrea/pull/6884), [@hangyan])
+- Support enabling multicast snooping for SecondaryNetwork. ([#7200](https://github.com/antrea-io/antrea/pull/7200), [@tnqn])
+- Support `k8s.v1.cni.cncf.io/networks` annotation removal to clean up Pod's secondary interfaces for SecondaryNetwork. ([#7119](https://github.com/antrea-io/antrea/pull/7119), [@wenqiq])
+- Allow defining static MAC addresses for SecondaryInterfaces for VLAN network. ([#7137](https://github.com/antrea-io/antrea/pull/7137), [@KMAnju-2021] [@rajnkamr])
+
+### Changed
+
+- Promote feature `AntreaIPAM` from Alpha to Beta. ([#7184](https://github.com/antrea-io/antrea/pull/7184), [@wenqiq])
```

Comment:
maybe let's not include this if there is no clear consensus regarding ongoing discussions

Reply:
Removed